packages/analytics-types/src/config.ts

```diff
@@ -83,7 +84,9 @@ export interface AttributionBrowserOptions {
   disabled?: boolean;
   excludeReferrers?: string[];
   initialEmptyValue?: string;
+  /** @deprecated New campaigns are now always tracked by default. Setting this to true also sets resetSessionOnNewCampaign to true. */
   trackNewCampaigns?: boolean;
+  /** @deprecated Use the new top level trackPageViews configuration instead. */
   trackPageViews?: boolean;
```

Comment:
these configs are technically not deprecated right? but are overwritten by plugin?

Reply:
Wouldn't deprecating them reduce confusion? since new implementers should use the plugin

Reply:
i guess i was thinking that for folks not interested in the plugin, these are not deprecated. plus the replacement `resetSessionOnNewCampaign` is not in core.

alternatively, i think we can just deprecate the entire attribution config and point them to the plugin

Reply:
Pointing them to the plugin is a good idea.

Reply:
Agree on deprecating entire config